lib/ansible/modules/windows/win_data_deduplication.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: 2019, rnsc(@rnsc) <github@rnsc.be>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: win_data_deduplication
+version_added: "2.9"
+short_description: Module to enable Data Deduplication on a volume.
+description:
+- This module can be used to enable Data Deduplication on a Windows volume.
+- The module will install the FS-Data-Deduplication feature (a reboot will be necessary).
+options:
+  drive_letter:
+    description:
+    - Windows drive letter on which to enable data deduplication.
+    required: yes
+    type: str
+  state:
+    description:
+    - Wether to enable or disable data deduplication on the selected volume.
+    required: yes
```

Comment:
If you have a `default` option then this is not true. I would just remove this line as `required: no` is the default.